packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -361,4 +361,50 @@ describe('HTS Precompile Acceptance Tests', async function () {
       expect(pauseStatusAfter).to.equal(false);
     });
   });
+
+  describe('HTS Precompile Default Status Tests', async function() {
+    it('should be able to get default freeze status for fungible token', async () => {
+      const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
+      
+      const txTokenDefaultFreezeStatus = await baseHTSContract.getTokenDefaultFreezeStatusPublic(HTSTokenContractAddress, {gasLimit: 1000000});
+      const responseCodeStatus = (await txTokenDefaultFreezeStatus.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode;
+      expect(responseCodeStatus).to.equal(TX_SUCCESS_CODE);
+
+      const freezeStatus = (await txTokenDefaultFreezeStatus.wait()).events.filter(e => e.event === 'TokenDefaultFreezeStatus')[0].args.defaultFreezeStatus;
+      expect(freezeStatus).to.equal(false);
+    });
+
+    it('should be able to get default freeze status for non fungible token', async () => {
+      const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
+      
+      const txTokenDefaultFreezeStatus = await baseHTSContract.getTokenDefaultFreezeStatusPublic(NftHTSTokenContractAddress, {gasLimit: 1000000});
+      const responseCodeStatus = (await txTokenDefaultFreezeStatus.wait()).events.filter(e => e.event === 'ResponseCode')[0].args.responseCode;
+      expect(responseCodeStatus).to.equal(TX_SUCCESS_CODE);
+
+      const freezeStatus = (await txTokenDefaultFreezeStatus.wait()).events.filter(e => e.event === 'TokenDefaultFreezeStatus')[0].args.defaultFreezeStatus;
+      expect(freezeStatus).to.equal(false);
+    });
+
+    it('should be able to get default kyc status for fungible token', async () => {
+      const baseHTSContract = new ethers.Contract(BaseHTSContractAddress, BaseHTSJson.abi, accounts[0].wallet);
+      
+      const txTokenDefaultKycStatus = await baseHTSContract.getTokenDefaultKycStatusPublic(HTSTokenContractAddress, {gasLimit: 1000000});
```

Comment:
nit: These 5 lines can be share by both the kyc methods

Reply:
I've prepared the kyc test here, starting with this default one. Will continue working on KYC test in issue #407